sdk/src/instruction.rs

```diff
@@ -58,6 +58,27 @@ pub enum InstructionError {
     CustomError(u32),
 }
 
+impl std::error::Error for InstructionError {
+    fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
+        None
+    }
+}
+
+impl std::fmt::Display for InstructionError {
```

Comment:
Can we just #derive Debug for InstructionError?

Reply:
we want to do that, too.  having Display is a requirement of std::error::Error, apparently